docs/index.md

```diff
@@ -3,12 +3,146 @@
+- an optional YAML file containing [experiment information](https://osf.io/xdkcy/wiki/Configuration%20YAML/) [![external link](img/external_link.png)](https://osf.io/xdkcy/wiki/Configuration%20YAML/)
+
+What we will  be doing:
+
+1. [Upload files and configure](#uploading-and-configuring-files) a server-side file storage definition
+2. Running the [Canopy Cover image-based workflow](#running-a-workflow)
+3. [Downloading the results](#downloading-the-results) to the local computer
+4. [Deleting](#delete-workflow-run) the workflow run
+
+Note that the geographic boundaries of the image don't need to match that of the plots.
+Only those plots that intersect the image will be processed.
+
+Additionally, the coordinate system of the image doesn't need to match that of the plots.
+Plot points are converted automatically to the image's geographic coordinate system.
+
+### Sample data
+
+We also provide an [archive of sample data](https://data.cyverse.org/dav-anon/iplant/projects/aes/cct/diag/sample-data/sample_website_data.tar.gz
+) [![external link](img/external_link.png)](https://data.cyverse.org/dav-anon/iplant/projects/aes/cct/diag/sample-data/sample_website_data.tar.gz) that can be downloaded and used in this Tutorial.
+Be sure to extract the files from this archive before uploading them.
+Uploaded archives are **not** usable by the system.
+
+The following `bash` commands can be used to download the archive and extract the files.
+
+```bash
+curl --output sample_website_data.tar.gz https://data.cyverse.org/dav-anon/iplant/projects/aes/cct/diag/sample-data/sample_website_data.tar.gz
+tar -xzf sample_website_data.tar.gz
+```
+
+### Uploading and configuring files
+
+On the main page of the site, click the `files` link under **Data Sources**.
+This will take you to the file storage definition page
+
+<img src="img/landing_click_files.png" width="450" alt="File storage definition link" aria-label="File storage definition link" title="Click on Files link"/>
+
+Next click the upload icon, select and upload the files to process.
+You may see an <img src="img/configure_files_uploading_files.png" width="90" alt="Uploading files" aria-label="Uploading files" title="Uploading files"/> message on the icon if it takes a few seconds to upload the files.
+You can also drag-and-drop files onto the upload location.
+
+<img src="img/configure_files_upload.png" width="450" alt="Click upload files icon" aria-label="Upload files link" title="Click or drag-and-drop to upload files"/>
+
+Next, select "Server Side" from the drop down on the upper right of the page, and click "New".
+
+<img src="img/configure_files_server_side.png" width="450" alt="Choose dropdown server-side then new" aria-label="Create new server-side definition" title="Create new server-side file configuration"/>
+
+This will display a new window showing the uploaded files.
+Change the name of the configuration if you want to.
+Select "OK" to complete the configuration and dismiss the window.
```

Comment:
when I try to rename the configuration I get an error "duplicate name found"

Reply:
@dlebauer I am unable to reproduce this. Can you provide the steps needed?

Reply:
- Create a folder holding a set of files
- Create a new one. Give it the same name as the one that already exists
- Error shows up

But this isn't really a problem. Will mark as resolved